libs/geometry/include/geometry/SurfaceOrientation.h

```diff
@@ -47,67 +47,59 @@ class SurfaceOrientation {
     public:
         Builder();
         ~Builder() noexcept;
+        Builder(Builder&&) = default;
+        Builder& operator=(Builder&&) = default;
 
         /**
          * These two attributes are required. They are not passed into the constructor to force
          * calling code to be self-documenting.
          * @{
          */
         Builder& vertexCount(size_t vertexCount) noexcept;
-        Builder& normals(const filament::math::float3*) noexcept;
+        Builder& normals(const filament::math::float3*, size_t stride = 0) noexcept;
         /** @} */
 
-        Builder& tangents(const filament::math::float4*) noexcept;
-        Builder& uvs(const filament::math::float2*) noexcept;
-        Builder& positions(const filament::math::float3*) noexcept;
+        Builder& tangents(const filament::math::float4*, size_t stride = 0) noexcept;
+        Builder& uvs(const filament::math::float2*, size_t stride = 0) noexcept;
+        Builder& positions(const filament::math::float3*, size_t stride = 0) noexcept;
 
         Builder& triangleCount(size_t triangleCount) noexcept;
         Builder& triangles(const filament::math::uint3*) noexcept;
         Builder& triangles(const filament::math::ushort3*) noexcept;
 
-        /**
-         * Supplying explicit stride values is optional, they each default to the size of their
-         * respective input attributes, e.g. sizeof(float3) for normals.
-         * @{
-         */
-        Builder& normalStride(size_t numBytes) noexcept;
-        Builder& tangentStride(size_t numBytes) noexcept;
-        Builder& uvStride(size_t numBytes) noexcept;
-        Builder& positionStride(size_t numBytes) noexcept;
-        /** @} */
-
         /**
          * Generates quats or panics if the submitted data is an incomplete combination.
          */
-        SurfaceOrientation* build();
+        SurfaceOrientation build();
 
     private:
         OrientationBuilderImpl* mImpl;
         Builder(const Builder&);
         Builder& operator=(const Builder&);
     };
 
-    static void destroy(SurfaceOrientation* so);
+    SurfaceOrientation(const SurfaceOrientation&) = default;
```

Comment:
SurfaceOrientation should also provide a move ctor / operator.

Reply:
Done.